tern/utils/container.py

```diff
@@ -150,5 +150,5 @@ def extract_image_metadata(image_tag_string):
         os.remove(temp_tarfile)
         if not os.path.exists(temp_path):
             raise IOError('Unable to untar Docker image')
-    except docker.errors.APIError:
+    except docker.errors.APIError:  # pylint: disable=try-except-raise
```

Comment:
See above comment.

Reply:
Since you have
```python
    try:
        ...
    except ...:
        raise
```
The try/catch doesn't really serve any purpose, all exceptions raised will bubble up.  I was interpreting it as a place-holder allowing you add a log message in special cases?

Reply:
The try-except here is to make sure tern exits gracefully if for some reason an API call to Docker doesn't work. This is something that is littered throughout the code as tern uses syscalls via `subprocess` quite liberally. Rather than squash the pylint error for these try-except-raises, I would like to find the "right" way to write it.

Reply:
Perhaps fixing the try-except-raises linting errors falls under technical debt. I'll create an issue for it so it's not overhead for just clearing pylint errors.

Reply:
OK, sounds good.  Will also try thinking about a clean pattern.